src/main/java/org/team1540/robot2023/commands/auto/Auto2PieceTaxiConeVision.java

```diff
@@ -0,0 +1,54 @@
+package org.team1540.robot2023.commands.auto;
+
+import com.pathplanner.lib.PathConstraints;
+import edu.wpi.first.wpilibj2.command.Command;
+import edu.wpi.first.wpilibj2.command.Commands;
+import org.team1540.robot2023.Constants;
+import org.team1540.robot2023.commands.arm.Arm;
+import org.team1540.robot2023.commands.arm.PivotCommand;
+import org.team1540.robot2023.commands.arm.ResetArmPositionCommand;
+import org.team1540.robot2023.commands.arm.RetractAndPivotCommand;
+import org.team1540.robot2023.commands.drivetrain.Drivetrain;
+import org.team1540.robot2023.commands.grabber.GrabberIntakeCommand;
+import org.team1540.robot2023.commands.grabber.WheeledGrabber;
+import org.team1540.robot2023.commands.vision.DriveToGamePiece;
+import org.team1540.robot2023.commands.vision.TurnToGamePiece;
+import org.team1540.robot2023.utils.AutoCommand;
+import org.team1540.robot2023.utils.PolePosition;
+import org.team1540.robot2023.utils.ScoringGridLocation;
+
+import java.util.List;
+
+public class Auto2PieceTaxiConeVision extends AutoCommand {
+    public Auto2PieceTaxiConeVision(Drivetrain drivetrain, Arm arm, WheeledGrabber intake, ScoringGridLocation.OuterGrid grid) {
+        List<Command> pathCommands = getPathPlannerDriveCommandGroup(drivetrain, grid.getPathName("2PieceTaxiCone"), new PathConstraints(4, 2), true);
```

Comment:
So uh, the path here needs to be different, because it can't go the last couple feet or the vision won't work